helper/consts/plugin_types.go

```diff
@@ -0,0 +1,59 @@
+package consts
+
+import "fmt"
+
+var PluginTypes = []PluginType{
+	PluginTypeUnknown,
+	PluginTypeCredential,
+	PluginTypeDatabase,
+	PluginTypeSecrets,
+}
+
+type PluginType int
```

Comment:
Not super important, but could we make this a uint32?

Reply:
Absolutely. Done and pushed.